File: api/api/src/main/kotlin/api/response/KontaktObrazecRes.kt
package api.response

import core.domain.Kontakt
import core.domain.Oseba
import core.domain.Sporocilo
import kotlinx.serialization.Serializable

@Serializable
data class KontaktObrazecRes(
    val oseba: Oseba,
    val telefon: Kontakt,
    val email: Kontakt,
    val sporocila: List<Sporocilo>,
)

File: app/src/app/core/services/api/models/kontakt-obrazec-res.ts
/* tslint:disable */
/* eslint-disable */
import { Kontakt } from './kontakt';
import { Oseba } from './oseba';
import { Sporocilo } from './sporocilo';
export interface KontaktObrazecRes {
  email: Kontakt;
  oseba: Oseba;
  sporocila: Array<Sporocilo>;
  telefon: Kontakt;
}
